Conversions/IntToRoman.java

```diff
@@ -0,0 +1,101 @@
+public class IntToRoman {
+    
+    	int value(int toPret){

```

Comment:
This methods you can write as ```static```. Then you don't need the instanciate a object of type ```IntToRoman``` and you can use this method right away in you main method.